config/config_test.go

```diff
@@ -336,6 +336,35 @@ func TestConcurrentTransfersNegativeValue(t *testing.T) {
 	assert.Equal(t, 3, n)
 }
 
+func TestBasicTransfersOnlySetValue(t *testing.T) {
+	config := &Configuration{
+		gitConfig: map[string]string{
+			"lfs.basictransfersonly": "true",
+		},
+	}
+
+	b := config.BasicTransfersOnly()
+	assert.Equal(t, true, b)
```

Comment:
I updated this suite to use the `github.com/stretchr/testify` package so you can use `assert.True` and `assert.False` here 😄